hc/accounts/migrations/0001_initial.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
- `.gitignore` has `migrations` folder. Why are you deleting the migration files from the repo?

Reply:
Because we noticed if someone clones the project with existing migrations and doesn't have the database with which the previous migration files were applied, then when he/she runs migrations all tables wont be created. 

To handle that situation we chose to exclude migrations files so that when someone pulls the latest changes they will have to run `makemigrations` and `migrate` commands so that the new changes in the model is applied.

Reply:
On second thought, we've decided to re-push the migrations. Our decisions was informed by consultations from our peers. 

These articles also influenced our decision.
https://stackoverflow.com/questions/32376092/why-there-is-need-to-push-django-migrations-to-version-control-system

https://stackoverflow.com/questions/33784362/django-migrations-best-practice